app/queries/find_cloud_fixity_failures.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+class FindCloudFixityFailures
+  def self.queries
+    [:find_cloud_fixity_failures]
+  end
+
+  attr_reader :query_service
+  delegate :resource_factory, :run_query, to: :query_service
+  delegate :orm_class, to: :resource_factory
+  def initialize(query_service:)
+    @query_service = query_service
+  end
+
+  def query(order_by_property: "updated_at", order_by: "ASC")
+    <<-SQL
+      SELECT res.* FROM orm_resources AS res WHERE NOT EXISTS (
+        SELECT a.* FROM orm_resources AS a WHERE
+          a.metadata @> jsonb_build_object('status', ARRAY['SUCCESS']) AND
+          a.metadata->'resource_id' = res.metadata->'resource_id' AND
+          a.metadata->'child_id' = res.metadata->'child_id' AND
+          a.metadata->'child_property' = res.metadata->'child_property' AND
+          a.updated_at >= res.updated_at
```

Comment:
This'll probably go faster if you add an `internal_resource` check, otherwise it'll table scan.